NeuroID/NeuroIDClass/Extensions/NIDAdvancedDevice.swift

```diff
@@ -110,7 +102,11 @@ public extension NeuroID {
     @objc internal static func captureAdvancedDevice(
         _ shouldCapture: [Bool] = [NeuroID.isAdvancedDevice]
     ) {
-        captureLOGEvent("isAdvancedDevice setting: \(shouldCapture)", level: "INFO")
+        let logEvent = NIDEvent(type: .log)
```

Comment:
NIT: we have a log message event init

NIDEvent(type:.log, level:String, m:String)

Reply:
using suggested method signature